web/src/components/v2/Activities.js

```diff
@@ -0,0 +1,137 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import ActivityDescription from './ActivityDescription';
+import ContractorResources from './ContractorResources';
+import CostAllocation from './CostAllocation';
+import Expenses from './Expenses';
+import Section from './Section';
+import SectionTitle from './SectionTitle';
+import StatePersonnel from './StatePersonnel';
+import Collapsible from '../Collapsible';
+import FormActivityGoals from '../FormActivityGoals';
+import FormActivitySchedule from '../FormActivitySchedule';
+import { STANDARDS, activityDisplay } from '../../util';
+
+const Activities = ({
+  activities,
+  addActivity,
+  editActivityChecks,
+  editActivityText
+}) => (
+  <Section>
+    <SectionTitle>Program Activities</SectionTitle>
+    <Collapsible title="Activity List">
+      <div className="mb2">
+        {activities.map((a, i) => (
+          <div key={a.id} className="flex items-center mb1">
+            <div className="mr1 bold mono">{i + 1}.</div>
+            <div className="mr1 col-4">
+              <input
+                type="text"
+                className="col-12 input m0"
+                name={`${a.id}.name`}
+                value={a.name}
+                onChange={editActivityText}
+              />
+            </div>
+            <div>
+              {['HIT', 'HIE', 'MMIS'].map(aType => (
+                <label key={aType} className="mr1">
+                  <input
+                    type="checkbox"
+                    name={`${a.id}.type`}
+                    value={aType}
+                    checked={a.type.includes(aType)}
+                    onChange={editActivityChecks}
+                  />
+                  {aType}
+                </label>
+              ))}
+            </div>
+          </div>
+        ))}
+      </div>
+      <button
+        type="button"
+        className="mt2 btn btn-primary"
+        onClick={addActivity}
+      >
+        Add activity
+      </button>
+    </Collapsible>
+
+    {activities.map((a, i) => (
+      <Collapsible
+        key={a.id}
+        title={`Program Activities › ${activityDisplay(a, i + 1)}`}
+        bgColor="darken-1"
+      >
+        <Collapsible title="Activity Description">
+          <ActivityDescription activity={a} />
+        </Collapsible>
+
+        <Collapsible title="Needs and Objectives">
+          <FormActivityGoals form={`activity-${a.id}-goals`} />
```

Comment:
Eek.  I get it, though.

Reply:
this `FormActivityGoals` will be gone entirely soon enough